flocker/node/agents/test/test_blockdevice.py

```diff
@@ -1748,3 +1934,180 @@ def get_mounts():
         for mount in mounts:
             device_path, mountpoint, filesystem_type = mount.split()[:3]
             yield FilePath(device_path), FilePath(mountpoint), filesystem_type
+
+
+# def _make_resize_dataset():
+#     """
+#     Make a ``ResizeBlockDeviceDataset`` instance for
+#     ``make_state_change_tests``.
+#     """
+#     return ResizeBlockDeviceDataset(
+#         volume=_ARBITRARY_VOLUME,
```

Comment:
I had to change `DestroyBlockDeviceDataset` to accept a `dataset_id`.  I suspect the same will be necessary for `ResizeBlockDeviceDataset`.